### ISSUE TITLE
Add learning rate decay to MoCo training funtion

### ISSUE THREAD

**Issue by nathanin** [nathanin/micron2#1]
https://github.com/tensorflow/tensorflow/issues/37862
